contents/blog/guardrail-metrics.md

```diff
@@ -0,0 +1,103 @@
+
+> **Secondary vs guardrail metric:** Guardrail metrics are a subset of secondary metrics. A secondary metric is a non-goal metric you monitor for either positive or negative impacts in A/B tests. Guardrail metrics generally monitor for **negative** impacts.
+
+## Why use guardrail metrics
+
+Guardrail metrics enable you to protect your overall experience and key metrics while optimizing parts of your product with A/B tests
+
+As a developer, using guardrail metrics provides benefits like:
+
+- More confidence A/B tests aren’t causing problems. You spend less time checking for the negative impact of an experiment.
+
+- A more accurate source of issues than bug reports or general metrics. Developers don’t need to guess if an A/B test is causing an issue, guardrail metrics can tell them.
+
+- More confidence in the results of the experiment knowing it doesn’t have broader negative effects.
+
+- Promoting a culture of experimentation knowing A/B tests are safer to do. Creates trust between teams knowing other teams’ A/B tests won’t affect them.
+
+### Why Airbnb uses guardrail metrics
+
+[Airbnb](https://medium.com/airbnb-engineering/designing-experimentation-guardrails-ed6a976ec669) relies on guardrails to enable teams to run A/B tests while keeping the overall experience strong. They found A/B tests often positively impact one team’s metrics, while negatively impacting another. For example, if they didn’t display house rules, they saw an increase in bookings but a decrease in ratings.
+
+When a guardrail metric triggers, it escalates and stakeholders decide whether to continue the test or not. Guardrails at Airbnb flag ~25 experiments per month for review. 80% of these eventually launch after stakeholder discussion. They stop ~5 per month before launch, which means they prevent 5 potential negative impacts.
+
+![](../images/blog/guardrail-metrics/trigger.png)
+
+## What guardrail metrics should you pick?
+
+As a general guideline, you should pick metrics important to the entire product or company. These are often [north star](/blog/north-star-metrics) or [product health](/blog/product-health-metrics) metrics. Specifically, Airbnb breaks guardrail metrics into three categories
+
+- Key business metrics
+- User experience metrics
+- Strategic priority metrics
+
+It is important to have a mix of metrics from each category to ensure a broad enough coverage of potential issues. It is also important to not have too many metrics. Adding one increases the likelihood of false positives, which limits the throughput of experiments. For example, an [A/A test](/tutorials/aa-testing) with a significance level of 0.05 would expect to cause the following amount of false positives:
+
+- With 3 guardrail metrics, you would have at least one false alert approximately 14% of the time.
+- With 10 guardrail metrics, approximately 40% of the time.
+- With 25 guardrail metrics, approximately 73% of the time.
```

Comment:
```suggestion
You need a mix of metrics from each category to ensure you don't miss potential issues. Just remember that each guardrail metric you add also increases the likelihood of false positives. 

For example, an [A/A test](/tutorials/aa-testing) with 3 guardrail metrics, and a significance level of 0.05, would cause a false alert approximately 14% of the time. The risk increases to 40% with 10 guardrail metrics, and a massive 73% with 25 guardrail metrics.

More false positives means you run fewer tests, limiting the pace of improvements. 
```

Could we recommend a sensible number of guardrail metrics per test?

Reply:
This seems easier to read than the bullet points to me.